gprofiler/utils/__init__.py

```diff
@@ -149,6 +153,8 @@ def start_process(
             # explicitly remove our directory from LD_LIBRARY_PATH
             env = env if env is not None else os.environ.copy()
```

Comment:
@Jongy @d3dave looks like when called with `env`, we simply don't pass `TMPDIR`. I suppose it's not the intended behavior, right?

Reply:
Indeed not the intended behavior, although the only process we executed that I expect to make use of TMPDIR is `PyPerf`.

Regardless - it's a bug - you're welcome to fix, but move it to a separate PR please.

Reply:
https://github.com/Granulate/gprofiler/pull/922